### ISSUE TITLE
bug(Chips): Screen reader does not announce proper selected state when multiselect enabled

### ISSUE THREAD

**Issue by striped-s0cks** [angular/components#27053]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When using Chips component that allows multiple options to be selected, screen reader does not announce the state correctly and causes confusion for customers. Without `multiple` attribute screen reader announces the "selected/not selected" state as expected.

### Reproduction

Steps to reproduce:
1. Open basic example from the Chips doc:
![image](https://user-images.githubusercontent.com/10143921/236651172-75b9d303-f1ca-4fe8-90c7-2d975d14d0af.png)

2. Add `multiple` attribute to `mat-chip-listbox` to allow multiselect.
```
<mat-chip-listbox multiple aria-label="Fish selection">
  <mat-chip-option>One fish</mat-chip-option>
  <mat-chip-option>Two fish</mat-chip-option>
  <mat-chip-option color="accent" selected>Accent fish</mat-chip-option>
  <mat-chip-option color="warn">Warn fish</mat-chip-option>
</mat-chip-listbox>
```

3. Turn on a screen reader and navigate between chip options.

### Expected Behavior

When multiselect enabled, when clicking/tabing on a chip, screen reader announces:

- (when option selected) "{label} selected"
- (when option not selected) "{label} not selected"

### Actual Behavior

When multiselect enabled, when clicking/tabing on a chip, screen reader announces:

On MacOS + VoiceOver:
- (when option selected) "{label} selected, selected"
- (when option not selected) "{label} not selected, selected"

On Linux + Orca:
- (when option selected) "{label}"
- (when option not selected) "{label} not selected"

![image](https://user-images.githubusercontent.com/10143921/236651100-b67ed0d2-d84e-47d9-9df3-387fabefb302.png)

### Environment

Not a custom env, able to reproduce in basic example on StackBlitz in Chrome:
![image](https://user-images.githubusercontent.com/10143921/236651165-6bb42b01-1e5a-4638-b893-a23c3542ed59.png)

**Comment by amysorto** [comment]
This seems fine for MacOS + VoiceOver, since the correct state is being read aloud. I'm closing this bug because [we don't support Orca](https://github.com/angular/components#browser-and-screen-reader-support).

**Comment by striped-s0cks** [comment]
Hey @amysorto, thanks for posting the list of supported tools.
Regarding MacOS + Chrome - are you saying that the current way SR announces state ("not selected, selected") is expected? Isn't it confusing?

![image](https://github.com/angular/components/assets/10143921/1379e3ab-9eec-474c-b47b-c33dda914edb)
